examples/Chair/dodo_Chair.py

```diff
@@ -0,0 +1,55 @@
+import datetime
+import logging
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+from doit import create_after, get_var
+from doit.task import clean_targets
+from doit.tools import config_changed
+
+from fenicsxconcrete.util import ureg
+
+from amworkflow.geometry import GeometryCenterline
+from amworkflow.meshing import MeshingGmsh
+from amworkflow.gcode import PowderbedCodeFromSTL
+from amworkflow.simulation import SimulationFenicsXConcrete
+
+# > doit -f <filename>   # for execution of all task
+# > doit -f <filename> s <taskname> # for specific task
+# > doit -f <filename> clean # for deleting task output
+
+logging.basicConfig(level=logging.INFO)
+
+# define required parameters
+params_print = {  # print parameters
+    #"stl_unit": 1000,       # units in your stl-file (how much of your unit is 1m, i.e. mm units would be factor 1000)
```

Comment:
These are old parameters or? If yes delete and add something like  'standard': 'PowderBedBAM' # default 
are there no parameters you could vary like velocity?